src/cache/cache_las.c

```diff
@@ -1428,6 +1440,9 @@ __wt_find_lookaside_upd(
 	ref = cbt->ref;
 	las_pageid = ref->page_las->las_pageid;
 	session_flags = 0;		/* [-Werror=maybe-uninitialized] */
+	i = 0;
```

Comment:
initialization is suggested in the same order as the declaration.

Reply:
067612a

Reordered.